src/browser/main.coffee

```diff
@@ -77,14 +77,9 @@ setupAtomHome = ->
     atomHome = fs.realpathSync(atomHome)
   process.env.ATOM_HOME = atomHome
 
-setupCoffeeCache = ->
-  CoffeeCache = require 'coffee-cash'
-  cacheDir = path.join(process.env.ATOM_HOME, 'compile-cache')
-  # Use separate compile cache when sudo'ing as root to avoid permission issues
-  if process.env.USER is 'root' and process.env.SUDO_USER and process.env.SUDO_USER isnt process.env.USER
-    cacheDir = path.join(cacheDir, 'root')
-  CoffeeCache.setCacheDirectory(path.join(cacheDir, 'coffee'))
-  CoffeeCache.register()
+setupCompileCache = ->
+  compileCache = require('../compile-cache')
```

Comment:
>  In my opinion, we should try to get that down, but not by maintaining what is now a completely duplicated code-path.

Yeah, I'm totally on board with de-duplication but there were only a couple lines of duplication here between the browser/render processes right? Just the part that sets the cache directory path and checks for the sudo case?

A 28ms startup time increase (14ms on the browser side, 14ms presumably on the render side) does seem a bit excessive and something to possibly investigate further before merging.